ntbs-integration-tests/TestRunnerBase.cs

```diff
@@ -61,17 +59,30 @@ protected string FullErrorMessage(string validationMessage)
         }
 
         protected async Task<HttpResponseMessage> SendFormWithData(IHtmlDocument document, Dictionary<string, string> formData)
+        {
+            return await SendFormWithData(document, formData, null);
+        }
+
+        protected async Task<HttpResponseMessage> SendFormWithData(
+            IHtmlDocument document,
+            Dictionary<string, string> formData,
+            string postRoute)

```

Comment:
Could we just have one method SendFormWithData with a default value null for postRoute, rather than defining two?

Reply:
Absolutely, good call